cea/plots/demand/energy_balance.py

```diff
@@ -9,6 +9,14 @@
 import pandas as pd
 import numpy as np
 
+__author__ = "Gabriel Happle"
+__copyright__ = "Copyright 2018, Architecture and Building Systems - ETH Zurich"
+__credits__ = ["Gabriel Happle","Jimeno A. Fonseca"]
+__license__ = "MIT"
+__version__ = "2.8"
```

Comment:
i wonder how important the `__version__` attribute is and what it should mean.

Reply:
it's inconsistent among the files. sure. i'm totally fine with leaving it for now! but at some point in the future we'll have to come up with a good reason :-)